xpdacq/beamtime.py

```diff
@@ -168,6 +168,65 @@ def Tramp(dets, exposure, Tstart, Tstop, Tstep, *, md=None):
     yield from plan
 
 
+def T_list(dets, exposure, T_list):
+    """
+    Scan over the temperature controller in a list of user defined steps
```

Comment:
write a brief description of the function on the top line (same line as the """) then blank line, then a longer description if needed, then the arguments and what is returned.

I am not sure what this does, but let me take a guess here:

```
"""defines a flexible scan with user-specified temperatures

A frame is exposed for the given exposure_time at each of the
temperatures in the T_list

...

```